recipes/westpa/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "westpa" %}
+{% set version = "2017.10" %}
+{% set sha256 = "3129dd1d651c4fe542599f2d5ecdbff09e5d704ea8c97c8ae7dca7b0bd4598a6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/westpa/westpa/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: './setup.sh && cp -r $SRC_DIR $PREFIX'
```

Comment:
What all is getting copied in this `cp` step?

Reply:
Basically westpa builds a bunch of cython modules in-place. Previously it was always run from the source. A user would clone the repo or grab the tarball, run the setup script and then point environment variable to the location and then westpa would adjust the `PYTHONPATH`. So it was never a proper python package that was installed into `site-packages`. With that setup in mind, we needed to but it somewhere so it seemed like copying the directory into `~/miniconda/env/xxx/westpa-yyy` (for example) was the best solution.

Reply:
What if we move it to the location where we want to live and then build it there? That should allow `conda-build` to do much of its magic detecting paths. Hopefully should save you some maintenance burden. ;)

Reply:
In that scheme, how would you do it?

Reply:
I guess copy all of the source to the directory in the `PREFIX` that we want to build in. Build the package. Then remove any content we don't want distributed (e.g. `.o` files).